formulaic/static/admin/formulaic/ember-formulaic/README.md

```diff
@@ -7,51 +7,51 @@ A short introduction of this app could easily go here.
 
 You will need the following things properly installed on your computer.
 
-* [Git](https://git-scm.com/)
-* [nvm](https://github.com/creationix/nvm)
-* [Node.js](https://nodejs.org/) (with NPM)
-* [Bower](https://bower.io/)
-* [Ember CLI](https://ember-cli.com/)
-* [PhantomJS](http://phantomjs.org/) - TK: for testing
+- [Git](https://git-scm.com/)
+- [Node.js](https://nodejs.org/) (with npm)
+- [Ember CLI](https://cli.emberjs.com/release/)
+- [Google Chrome](https://google.com/chrome/)
 
 ## Installation
 
-* `nvm use`
-* `npm install`
-* `bower install`
+- `git clone <repository-url>` this repository
+- `cd ember-formulaic`
+- `ember g ember-cli-sass`
+- `npm install`
 
 ## Running / Development
 
-* `nvm use`
-* `ember serve`
+- `npm run start`
+- Visit your app at [http://localhost:4200](http://localhost:4200).
+- Visit your tests at [http://localhost:4200/tests](http://localhost:4200/tests).
 
 ### Code Generators
 
 Make use of the many generators for code, try `ember help generate` for more details
 
-### Running Tests - TK
+### Running Tests
 
-* `ember test`
-* `ember test --server`
+- `npm run test`
+- `npm run test:ember -- --server`
 
-### Building - TK: I generally don't differentiate the build created by `ember serve` from `ember build`
+### Linting
 
-* `ember build` (development)
-* `ember build --environment production` (production)
+- `npm run lint`
+- `npm run lint:fix`
+
+### Building
+
+- `npm exec ember build` (development)
+- `npm run build` (production)
```

Comment:
Can you elaborate on these instructions? It's not actually too clear what I actually need to do to build the ember project?

We're modifying the dockerfile to install node, but then don't do anything else? Is the intention to be able to build the project inside of docker or not?

Can I have a more step by step guide to building the project?